docs/releases/5.2.txt

```diff
@@ -269,7 +269,8 @@ PostgreSQL 14 and higher.
 Miscellaneous
 -------------
 
-* ...
+* :attr:`~django.core.mail.EmailMultiAlternatives.alternatives` should only be added to using :meth:`~django.core.mail.EmailMultiAlternatives.attach_alternative`.
+* ``EmailMessage.attachments`` should only be added to using the ``attach`` or ``attach_file`` methods.
```

Comment:
Requiring dedicated methods to modify `attachments` somehow feels like a bigger and potentially-breaking change than requiring them for `alternatives`. (Especially since `attachments` has always been documented as "can be set at any time," while the `alternatives` attr was not officially documented.)

Your code as currently implemented still supports unnamed tuples in `attachments`. I'd lean toward keeping that for backwards compatibility, and removing the second bullet here.

(But I could be convinced otherwise. As someone who maintains EmailBackends, I'd love to count on `attachments` being a list of `EmailAttachment`. I don't have a good sense for how much existing code this might break.)

Reply:
Agreed, we can remove the bullet
We should probably update the changed in 5.2 note around attachments from
```
In older versions, tuple items of ``attachments`` were regular tuples,
as opposed to :class:`~django.core.mail.EmailAttachment` named tuples.
```
to something like
```
Support for :class:`~django.core.mail.EmailAttachment` items of 
``attachments`` were added.
```